src/mono/wasm/runtime/loader/assets.ts

```diff
@@ -230,11 +230,14 @@ export async function start_asset_download(asset: AssetEntryInternal): Promise<A
         // second attempt only after all first attempts are queued
         await loaderHelpers.allDownloadsQueued.promise;
         try {
+            mono_log_debug(`Retrying download '${asset.name}'`);
```

Comment:
Would making this a trace message be better? Just wondering if this users would see this unnecessarily. Do people usually have the debug messages turned on, maybe when debugging their apps? Just wondering aloud.

Reply:
I think debug is ok. The rest of the similar messages are debug and I think this one of the same importance.